src/util/iroha-util.js

```diff
@@ -358,32 +321,31 @@ function getAllUnsignedTransactions (accountId) {
 /**
  * wrapper function of commands
  * @param {String} privateKey
- * @param {Function} buildQuery
- * @param {Number} timeoutLimit timeoutLimit
+ * @param {Object} tx transaction
+ * @param {Number} timeoutLimit
  */
 function command (
   privateKey = '',
-  buildTx = function () {},
+  tx,
   timeoutLimit = DEFAULT_TIMEOUT_LIMIT
 ) {
   let txClient, txHash
 
   return new Promise((resolve, reject) => {
-    const tx = buildTx()
-    const keys = crypto.convertFromExisting(
-      crypto.fromPrivateKey(privateKey).publicKey().hex(),
-      privateKey
-    )
-    const protoTx = makeProtoTxWithKeys(tx, keys)
+    let txToSend = flow(
+      (t) => txHelper.addMeta(t, { creatorAccountId: cache.username }),
+      (t) => txHelper.sign(t, cache.key)
```

Comment:
It should use privateKey, not cache.key, because cache.key will be removed in the future and it will become to take privateKey every time.

Reply:
Oh, I did it by accident, since copypasted from queries. Will fix.